File: Csoport_14_00/3. hét/Movie/Movie.java
import java.util.Scanner;

public class Movie {
    private String title;
    private int releaseYear;
    private double score;

    public Movie(String title, int releaseYear, double score) {
        this.setTitle(title);
        this.setReleaseYear(releaseYear);
        this.setScore(score);
    }

    public void setTitle(String value) {
        this.title = value;
    }

    public String getTitle() {
        return this.title;
    }

    public void setReleaseYear(int value) {
        this.releaseYear = value;
    }

    public int getReleaseYear() {
        return this.releaseYear;
    }

    public double getScore() {
        return score;
    }

    public void setScore(double value) {
        if (value < 0) {
            this.score = 0;
        }
        else if (value > 10) {
            this.score = 10;
        }
        else {
            this.score = value;
        }
    }

    @Override
    public String toString() {
        return String.format("Movie(%s (%d), %.1f)", this.getTitle(), this.getReleaseYear(), this.getScore());
    }
}

File: Csoport_14_00/12. hét/Osztalyok/Movie.cs
namespace Osztalyok;

public class Movie
{
    private string _title;
    // private int _year;
    private double _score;

    public Movie(string title, int year, double score)
    {
        _title = title;
        //_year = year;
        Title = title;
        _score = score;
    }

    /*public void SetTitle(string value)
    {
        this._title = value;
    }

    public string GetTitle()
    {
        return this._title;
    }*/

    public string Title
    {
        get
        {
            return this._title;
        }
        set
        {
            this._title = value;
        }
    }
    
    public int Year { get; set; }

    public override string ToString()
    {
        return string.Format("Movie({0} ({1}), {2})", Title, Year, _score);
    }
}